pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1400,6 +1523,78 @@ func TestSingleConstraint(t *testing.T) {
 				"node-b": framework.Unschedulable,
 			},
 		},
+		{
```

Comment:
I'd like to refactor this test and the next one like this:

- node-a carries TaintNodeUnschedulable, with 2 existing pods
- node-b has 1 existing pods
- (node-a and node-b is in zone1)
- node-c has 2 existing pods
- (node-c is in zone2)

The testing pod sets the topologyspreadconstraint in zone:

1. test1: the pod doesn't carry the matched toleration, it's expected to be scheduled to node-b (and node-a, in UT only)
1. test2: the pod carries the matched toleration, it's expected to be scheduled to node-c